hack/docker-builder/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM fedora:28
+
+ENV LIBVIRT_VERSION 4.2.0
+
+COPY fedora-virt-preview.repo /etc/yum.repos.d/fedora-virt-preview.repo
+
+RUN dnf -y install libvirt-devel-${LIBVIRT_VERSION} make git mercurial sudo gcc findutils gradle rsync-daemon rsync qemu-img protobuf-compiler && \
+    dnf -y clean all
+
+ENV GIMME_GO_VERSION=1.10
+
+RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
+
+ENV GOPATH="/go" GOBIN="/usr/bin"
+
+ADD rsyncd.conf /etc/rsyncd.conf
+
+RUN \
+    mkdir -p /go && \
+    source /etc/profile.d/gimme.sh && \
+    go get github.com/mattn/goveralls && \
+    go get -u github.com/Masterminds/glide && \
+    go get golang.org/x/tools/cmd/goimports && \
+    git clone https://github.com/mvdan/sh.git $GOPATH/src/mvdan.cc/sh && \
+    cd /go/src/mvdan.cc/sh/cmd/shfmt && \
+    git checkout v2.5.0 && \
+    go get mvdan.cc/sh/cmd/shfmt && \
+    go install && \
+    go get -u github.com/golang/mock/gomock && \
+    go get -u github.com/rmohr/mock/mockgen && \
+    go get -u github.com/rmohr/go-swagger-utils/swagger-doc && \
```

Comment:
Not that well versed yet, you'll have to be more specific

Reply:
Please go over the packages you install here and remove those not used in ovs-cni (e.g.  swagger doc, but probably more)

Reply:
Again, please be more specific. Will naively going through imports in pkg work? Do I need to check what is present in vendor? I'm not asking for the definition of keeping dependencies to a minimum... Since you wrote the plugin itself, you could obviously hint towards what I can conclude as obsolete and what is not